portfolio/src/main/client/src/util/html.ts

```diff
@@ -0,0 +1,153 @@
+import type {Arrunk} from './types';
+
+type InterpolationValue = Arrunk<string | Node | null | undefined> | EventListener;
```

Comment:
Why does this accept an event listener?

Reply:
Accepts an event listener to allow the `@click` handler and similar. Since you said I should not have this feature, I will remove that.